apps/framework-cli/src/infrastructure/processes/kafka_clickhouse_sync.rs

```diff
@@ -86,91 +81,6 @@ impl SyncingProcessesRegistry {
         self.to_topic_registry.insert(key, syncing_process.process);
     }
 
-    pub async fn start_all(
-        &mut self,
-        framework_object_versions: &FrameworkObjectVersions,
-        version_syncs: &[VersionSync],
-        metrics: Arc<Metrics>,
-    ) -> Result<(), rdkafka::error::KafkaError> {
-        info!("<DCM> Starting all syncing processes");
-
-        let kafka_config = self.kafka_config.clone();
-        let clickhouse_config = self.clickhouse_config.clone();
-
-        let available_topics: HashSet<String, RandomState> =
-            HashSet::from_iter(fetch_topics(&kafka_config).await?);
-
-        // Spawn sync for the current and old models
```

Comment:
That feels like it should not get deleted

Reply:
The function isn't called by anything else, and I also checked prod logs for the absence of "Starting all syncing processes"

Reply:
Yeah - double checked as well - that was surprising to me but it looks we had dropped some DCM functionality already